vue/components/ui/molecules/table/table.vue

```diff
@@ -430,7 +430,9 @@ export const Table = {
             }
 
             const items = [...this.itemsWithIndex];
-            return this.sortMethod(items, this.sortData, this.reverseData);
+            const sortedItems = this.sortMethod(items, this.sortData, this.reverseData);
+            this.$emit("update:sorted-items", sortedItems);

```

Comment:
Don't do these kind of operations in a computed variable.

Add a watcher for the `sortedItems` computed variable and then do the emit operation there